root/etc/cont-init.d/40-plex-first-run

```diff
@@ -6,7 +6,7 @@ if [ "${DEBUG,,}" = "true" ]; then
 fi
 
 # If the first run completed successfully, we are done
-if [ -e /.firstRunComplete ]; then
+if [ -e /tmp/.firstRunComplete ]; then
```

Comment:
If moved to another directory, this if should check for the file existing on `/` to account for those where it has already run in this container before this file was moved to another directory.

Reply:
This I do not understand. Upgrading from one container image to another (version upgrade) will definitively loose those file in `/.firstRunComplete`. There is definitely no edge case where this file would survive a container upgrade.
- If one would define a volume to `/`, plex would not work as the content of `/` would not be visible anymore before the entrypoint command would be executed
- You cannot upgrade a container from one image version to another image version without loosing data not living in a mapped volume

Reply:
Sorry, when I wrote this I was thinking of something else.